### DISCUSSION TITLE
Create a Data App from the Web - Observable Cloud

### DISCUSSION THREAD

**Discussion by gcoch** [observablehq/framework#1665] [Ideas]
It would we very useful to be able to create a Data App from the web, just like with Notebooks, which would then make it very easy to connect to an existing github repository.

**Comment by tophtucker** [comment]
I agree!! By chance I was already working on it, and it just went live; go to https://observablehq.com/select-workspace?next=projects-new and you can pick a GitHub repository to make a data app from it. Much faster than hopping back and forth between CLI and website.

https://github.com/user-attachments/assets/e8686bdf-00c8-49bf-8df7-68f43157444c

**Reply by gcoch** [reply]
Fantastic! I just saw it also. I'm seeing this message. What does it mean?

<img width="795" alt="Screenshot 2024-09-18 at 7 37 45 PM" src="https://github.com/user-attachments/assets/14cb0aa2-b83e-4d32-9d0b-7deaed7a4fc1">

We need to be able to edit the name of the App though.

**Reply by tophtucker** [reply]
Ah, are you still seeing that? It looks OK to me now:

<img width="329" alt="image" src="https://github.com/user-attachments/assets/77aba601-695d-454c-af12-ead12c83cb12">

“Not connected” means we had some downtime where our servers wouldn’t be available to immediately build your app. You don’t have to do anything; it’s our responsibility to get it back up fast. (Enterprise workspaces can set up on-premises environments, which you manage yourself, which lets you choose which environment you want to build an app in. But hopefully our uptime should be pretty good & getting better & you won’t have to think about it!)

**Reply by tophtucker** [reply]
As for editing the name of the app, that’s determined by the title in your app’s observablehq.config.js file; it’s updated whenever you deploy. (Good reminder, I’ve been meaning to add a tooltip to that effect.)

**Reply by gcoch** [reply]
Hi, I actually think that when you create the app from Github it takes the name of the repo and not the title.

**Reply by tophtucker** [reply]
Oh man you’re totally right! Well, I knew we _initially_ used the GitHub repo name as the data app name, but I thought it’d be overwritten every time you deploy. 

But in fact we only update the data app title from observablehq.config.js if you deploy manually from the command line (like running `npm run deploy` yourself). We never update the title during a cloud build. I think that’s a bug. I’m looking into it.

In the interim, if you want to change the title you’ll have to (1) make sure you have the right title in observablehq.config.js and then (2) do a manual local deploy.

**Reply by tophtucker** [reply]
@mythmon [fixed this](https://github.com/observablehq/framework/pull/1715) in Framework; it’ll be included in the next release, which could take a couple weeks idk, and then you’ll need to upgrade to the latest version for your titles to start updating during cloud builds. Annoying! Sorry! Feels like a basic thing for us to have missed, so thanks for raising this!

**Reply by gcoch** [reply]
Related to this it would be very useful to be able to enable change the schedules from the web instead of in the yaml file.
